### PR TITLE
Add BASK as Collateral Type

### PR DESCRIPTION
**Pull request by mhairimca**

### CODE REVIEW

File: Add BASK as a collateral type.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- **Author**   - *Shawn C. Hagenah (Hagz48)shawnhagenah99@yahoo.com*
+- **Status: Draft**
+- **Created:** *6/2/2021*
+- **Discourse Link:** *Insert link to discourse topic **after** it has been moved into draft UMIPs*
+
+## Summary (2-5 sentences)
+*The BASK token is the Governance token for the Basket DAO Protocol. Adding BASK as a price collateral currency for the creation of synthetic tokens is the purpose of this collateral UMIP.*
+
+## Motivation
+*Adding BASK as a collateral currency would allow Basket DAO community members the ability to create synthetic tokens for the creation of KPI options. The creation of these tokens using the UMA protocol, will not only allow for growth in the Basket DAO community but would also raise the current TVL of the UMA protocol thus benefitting both communities.  Basket DAO has also expressed interest in other UMA protocol products as well.*
```
Comment (akhanubis) [inline Add BASK as a collateral type.md:15]: Give the ability or allow to instead of allow the ability to

File: Add BASK as a collateral type.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Technical Specification
+
+**To accomplish this upgrade, the following changes need to be made:**
+
+* **The** *BASK* **address** **https://etherscan.io/address/0x44564d0bd94343f72e3c8a0d22308b7fa71db0bb **needs to be added to the collateral currency whitelist introduced in UMIP-8.** 
+
+* **A final fee of 5 BASK** *a minimum of approx 400 USD * in the proposed collateral currency* **needs to be added for** *BASK* **in the Store contract.**
+
+## Rationale
+*Adding BASK as a collateral type would allow the Basket DAO community the ability to create the KPI options that it has expressed it would like to mint. These options will not only help in the growth to the Basket DAO community, but the adding BASK as a collateral currency would also increase the UMA protocols TVL, thus benefitting both communities. The Basket DAO community has also expressed interest in creating Call options in the future. Thus adding the Basket DAO governance token BASK as a collateral type will benefit UMA both now, and into the future. Basket DAO while fairly new shows good liquidity and its value keeps on par with other similar coins.
```
Comment (akhanubis) [inline Add BASK as a collateral type.md:26]: Give the ability or allow to instead of allow the ability to

File: Add BASK as a collateral type.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- **Status: Draft**
+- **Created:** *6/2/2021*
+- **Discourse Link:** *Insert link to discourse topic **after** it has been moved into draft UMIPs*
+
+## Summary (2-5 sentences)
+*The BASK token is the Governance token for the Basket DAO Protocol. Adding BASK as a price collateral currency for the creation of synthetic tokens is the purpose of this collateral UMIP.*
+
+## Motivation
+*Adding BASK as a collateral currency would allow Basket DAO community members the ability to create synthetic tokens for the creation of KPI options. The creation of these tokens using the UMA protocol, will not only allow for growth in the Basket DAO community but would also raise the current TVL of the UMA protocol thus benefitting both communities.  Basket DAO has also expressed interest in other UMA protocol products as well.*
+
```
Comment (akhanubis) [inline Add BASK as a collateral type.md:16]: I'd add a sentence or two regarding it's current market cap and liquidity in exchanges for informational purposes considering that call options might be in the horizon.

**Review by akhanubis — COMMENTED** [review]
Looks overall good to me, apart from a few proposed minor changes.

File: Add BASK as a collateral type.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- **UMIP title:**  - Add *BASK* as collateral 
+- **Author**   - *Shawn C. Hagenah (Hagz48)shawnhagenah99@yahoo.com*
+- **Status: Draft**
+- **Created:** *6/2/2021*
+- **Discourse Link:** *Insert link to discourse topic **after** it has been moved into draft UMIPs*
+
+## Summary (2-5 sentences)
+*The BASK token is the Governance token for the Basket DAO Protocol. Adding BASK as a price collateral currency for the creation of synthetic tokens is the purpose of this collateral UMIP.*
+
+## Motivation
```
Comment (Reinis-FRP) [inline Add BASK as a collateral type.md:14]: Please provide some background information on BASK market cap, circulating supply, vesting/distribution schedules, etc.

File: Add BASK as a collateral type.md (hunk 1 above)
Comment (Hahz48) [reply Add BASK as a collateral type.md:15]: how do i get into it to make changes?

Comment (Hahz48) [reply Add BASK as a collateral type.md:15]: All right I got in. Not sure were exactly you would like that change made?

File: Add BASK as a collateral type.md (hunk 3 above)
Comment (Hahz48) [reply Add BASK as a collateral type.md:16]: Ok, I will add that in security consideratiins?

Comment (Hahz48) [reply Add BASK as a collateral type.md:16]: considerations? Market cap and liquid in motivation

File: Add BASK as a collateral type.md (hunk 1 above)
Comment (therealhexi) [reply Add BASK as a collateral type.md:15]: I believe he meant: Adding BASK as a collateral currency would `give` Basket DAO community members the ability to

File: Add BASK as a collateral type.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+**To accomplish this upgrade, the following changes need to be made:**
+
+* **The** *BASK* **address** **https://etherscan.io/address/0x44564d0bd94343f72e3c8a0d22308b7fa71db0bb **needs to be added to the collateral currency whitelist introduced in UMIP-8.** 
+
+* **A final fee of 5 BASK** *a minimum of approx 400 USD * in the proposed collateral currency* **needs to be added for** *BASK* **in the Store contract.**
+
+## Rationale
+*Adding BASK as a collateral type would allow the Basket DAO community the ability to create the KPI options that it has expressed it would like to mint. These options will not only help in the growth to the Basket DAO community, but the adding BASK as a collateral currency would also increase the UMA protocols TVL, thus benefitting both communities. The Basket DAO community has also expressed interest in creating Call options in the future. Thus adding the Basket DAO governance token BASK as a collateral type will benefit UMA both now, and into the future. Basket DAO while fairly new shows good liquidity and its value keeps on par with other similar coins.  
+
```
Comment (therealhexi) [inline Add BASK as a collateral type.md:27]: In addition to the benefits it brings, I would like to see some rationale on why BASK is a good collateral currency. For example, does it have enough circulating supply to sustain price manipulation? Is the currency liquid on multiple large exchanges so liquidators or disputers can buy back the tokens?

File: Add BASK as a collateral type.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+## Implementation
+
+**This change has no implementations  other than the afore mentioned governor transactions*
+
+## Security considerations
+*BASK is the governance token of the Basket DAO protocol.Its implementation as a collateral currency should pose no security threats other than normal market volatility. That being said $UMA holders should evaluate the ongoing cost and benefit of supporting this asset as collateral if liquidity concerns are identified. UMA holders should take note of the collaterals nature as liquidity if the collateral changes or if added robustness(Eg. via TWAPs are necessary to prevent market manipulation.*
```
Comment (therealhexi) [inline Add BASK as a collateral type.md:33]: Nit: ` ` between Basket DAO protocol.` `Its implementation.

I notice a few other places where you have multiple spaces, e.g. line 30 between "implementations" and "other". Please cross check the spacing of all text in this file

**Review by therealhexi — COMMENTED** [review]
Comments

File: Add BASK as a collateral type.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+
+
+## Headers
+- **UMIP #** -  *Leave blank - an UMIP no will be assigned at proposal stage*
```
Comment (smb2796) [inline Add BASK as a collateral type.md:4]:
```suggestion
- **UMIP #** -
```

File: Add BASK as a collateral type.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+- **UMIP title:**  - Add *BASK* as collateral 
+- **Author**   - *Shawn C. Hagenah (Hagz48)shawnhagenah99@yahoo.com*
+- **Status: Draft**
+- **Created:** *6/2/2021*
+- **Discourse Link:** *Insert link to discourse topic **after** it has been moved into draft UMIPs*
```
Comment (smb2796) [inline Add BASK as a collateral type.md:9]: Needs a discourse link

File: Add BASK as a collateral type.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+## Summary (2-5 sentences)
+*The BASK token is the Governance token for the Basket DAO Protocol. Adding BASK as a price collateral currency for the creation of synthetic tokens is the purpose of this collateral UMIP.*
```
Comment (smb2796) [inline Add BASK as a collateral type.md:12]: It'd be best to remove all of the italic formatting, as this is just a part of the price id template and not the expected formatting for the UMIP.

File: Add BASK as a collateral type.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+## Motivation
+*Adding BASK as a collateral currency would allow Basket DAO community members the ability to create synthetic tokens for the creation of KPI options. The creation of these tokens using the UMA protocol, will not only allow for growth in the Basket DAO community but would also raise the current TVL of the UMA protocol thus benefitting both communities.  Basket DAO has also expressed interest in other UMA protocol products as well.*
+
+## Technical Specification
+
+**To accomplish this upgrade, the following changes need to be made:**
+
+* **The** *BASK* **address** **https://etherscan.io/address/0x44564d0bd94343f72e3c8a0d22308b7fa71db0bb **needs to be added to the collateral currency whitelist introduced in UMIP-8.** 
+
+* **A final fee of 5 BASK** *a minimum of approx 400 USD * in the proposed collateral currency* **needs to be added for** *BASK* **in the Store contract.**
```
Comment (smb2796) [inline Add BASK as a collateral type.md:23]: Suggest an update to 4 BASK.

**Review by smb2796 — APPROVED** [review]
There are still some outstanding issues, but, as talked about on the community call, it seems like you're having some issues with github - so I'll resolve on my side

**Comment by Hahz48** [comment]
Yeah, im having alot of issues with github. I fixed the things and can see the commits but I have no idea were I am sending them. I tried all day yesterday, and before the call, no idea what I am doing. Also cannot figure out how to get from discourse to github. Also, what exactly do we need for UMIPs for BASK? What kind ,type how many ect. I'm pretty sure they will be losing intrest soon. I also have no clue on how to get price feeds for the Omnibus, I was under the assumption that the web address  to cryptowatch would be sufficient. I see some UMIPs with the price feeds showing some they are not, none have the same setup that is in the template. More than likely someone will have to finish jjc umip or just start all over again with it.Sent from my Galaxy
-------- Original message --------From: Sean Brown ***@***.***> Date: 6/8/21  10:10 AM  (GMT-06:00) To: UMAprotocol/UMIPs ***@***.***> Cc: Hahz48 ***@***.***>, Comment ***@***.***> Subject: Re: [UMAprotocol/UMIPs] Add BASK as Collateral Type (#314) 
@smb2796 approved this pull request.

There are still some outstanding issues, but, as talked about on the community call, it seems like you're having some issues with github - so I'll resolve on my side

**Pull request merged**